spec/index.bs

```diff
@@ -1577,13 +1577,15 @@ interface GPUCommandEncoder {
         GPUBufferSize size);
 
     void copyBufferToTexture(
-        GPUBufferCopyView source,
+        GPUBuffer source,
+        GPUTextureDataLayout sourceLayout,
         GPUTextureCopyView destination,
         GPUExtent3D copySize);
 
     void copyTextureToBuffer(
         GPUTextureCopyView source,
-        GPUBufferCopyView destination,
+        GPUBuffer destination,
```

Comment:
nit: I kinda like the symmetry between `GPUTextureCopyView` and `GPUBufferCopyView`, maybe `GPUBufferCopyView` could extend `GPUTextureDataLayout`?

Reply:
Interesting idea I haven't considered, let me try it